scipy/special/_cdflib.pyx

```diff
@@ -3630,8 +3630,8 @@ cdef inline (double, int, double) cdftnc_which3(
                                     0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
                                     0, 0, 0, 0, 0, 0, 0)
 
-    DS.small = 0
-    DS.big = 1.e6
+    DS.small = 1e-100
+    DS.big = 1.e10
```

Comment:
I changed here to be consistent with the documentation above.
https://github.com/scipy/scipy/blob/5c364a73ffb20b31b6afae5a18184ebff4758695/scipy/special/_cdflib.pyx#L3523-L3526
Probably the small value 0 makes the computation of `nctdtridf` problematics.